primeqa/pipelines/base.py

```diff
@@ -0,0 +1,72 @@
+from typing import Union, List,Any
```

Comment:
Looks like `primeqa/components/base.py`, but with different formatting and the `__hash__` functions removed?